server/game/cards/attachments/04/motherofdragons.js

```diff
@@ -15,13 +16,12 @@ class MotherOfDragons extends DrawCard {
             return false;
         }
 
-        if(this.kneeled || !player.cardsInPlay.any(card => {
-            return challenge.isParticipating(card) && card.hasTrait('Dragon');
-        })) {
+        if(!player.cardsInPlay.any(
+            card => challenge.isParticipating(card) && card.hasTrait('Dragon')
```

Comment:
This (and the if checking that there is a challenge above it) should be converted to a `condition` for the action.